cpu/stm32/kconfigs/Kconfig.clk

```diff
@@ -115,7 +119,115 @@ config CLOCK_HSISYS_DIV
     default 32 if CLOCK_HSISYS_DIV_32
     default 64 if CLOCK_HSISYS_DIV_64
     default 128 if CLOCK_HSISYS_DIV_128
-endif
+endif  # CPU_FAM_G0
+
+endif  # CPU_FAM_G0 || CPU_FAM_G4
+
+if CPU_FAM_L0 || CPU_FAM_L1
+choice
```

Comment:
Any special reason for making this a choice and translation instead of making `CLOCK_PLL_DIV` configurable with range? (in this case values are contiguous)
```
config CLOCK_PLL_DIV
    int "Main PLL division factor" if USE_CLOCK_PLL
    default 2
    range 2 4
```

Reply:
You are right, there's no need for a choice here.